validations/custom.validations.js

```diff
@@ -0,0 +1,15 @@
+const password = (value, helpers) => {
```

Comment:
- add an index.js in validators
- import all validators to index.js
- exports.module { /*all validators, can also be renamed here*/}